### PR TITLE
Weekly PR

### PR DESCRIPTION
**Pull request by EugeneButusov**
+ Docs #165 Remove links to library from unity in github
 + Feature #143 GPGS for Profile
 + Feature #161 Create download_deps script in all repos

### CODE REVIEW

**Comment by EugeneButusov** [comment]
@borisspektor @refaelos, it's ready to merge.

File: Soomla/Assets/Examples/MuffinRush/ExampleWindow.cs [hunk 1]
```diff
@@ -34,7 +34,7 @@ public class ExampleWindow : MonoBehaviour {
 	private static bool isVisible = false;
 	private bool isInit = false;
 
-	private Provider targetProvider = Provider.FACEBOOK;
```
Comment (borisspektor) [inline Soomla/Assets/Examples/MuffinRush/ExampleWindow.cs]: @EugeneButusov why did you change this?
Next time when you're testing please add a scenario to the example project, don't change it.
